### ISSUE TITLE
ENH: add more aka of for cdist

### ISSUE THREAD

**Issue by tupui** [scipy/scipy#14039]
Distance metrics have aliases like: `cityblock`, `cblock`, `cb` are all referring to the same metric. But this one is also commonly known as `manhattan` or `taxicab` distance.

We could add these and also things like `linf`, `l2`, etc.

cc @peterbell10

**Comment by tylerjereddy** [comment]
I'm not sure this is a specific enough proposal for change. You want to populate the `cdist`/`pdist` metric names with more aliases? Have we established that the benefits outweigh the drawbacks? Where does one draw the line on adding aliases?

> We could add these and also things like linf, l2, etc.

I'm not sure the benefit outweights the churn on adding `l2` on top of `euclidean` and `linf` on top of `chebyshev`.

**Comment by rgommers** [comment]
I also don't think this is necessarily a good idea. It certainly needs discussion, so removing the sprint label.

**Comment by mckib2** [comment]
If there were any change here, I would support doc-deprecating aliases and choosing a single set of metric labels

**Comment by peterbell10** [comment]
The metric aliases seem weird to me. AFAIK they aren't documented anywhere, and instead of being actual alternative names they're mainly just shortened forms of the same name. Like `manhattan` would makes sense as it's another name for the metric, but allowing `cityblock` to be shortened to `cblock` or `cb` just seems arbitrary to me.

**Comment by rgommers** [comment]
> If there were any change here, I would support doc-deprecating aliases and choosing a single set of metric labels

+1 this would be good

**Comment by tupui** [comment]
Indeed, looks like we need to discuss this more. I was just always surprised to not be able to use `manhattan` for instance although it's a very popular name. Then reading what you all said, the issue would be more to define a proper set of allowed keyword.

Then can we list the ones we would like to keep (I also agree that we should not have arbitrarily shortened versions like `cb`) and then we can plan how we move forward (deprecation cycle, etc.)
